activiti-cloud-audit-service/activiti-cloud-services-audit-jpa/src/test/java/org/activiti/cloud/services/audit/jpa/controller/EventsEngineEventsAdminControllerIT.java

```diff
@@ -62,15 +62,15 @@
 @EnableSpringDataWebSupport
 @AutoConfigureMockMvc
 @Import({ AuditAPIAutoConfiguration.class, AuditJPAAutoConfiguration.class, AlfrescoWebAutoConfiguration.class })
-public class EventsEngineEventsAdminControllerIT {
+class EventsEngineEventsAdminControllerIT {
 
     private static final String HEADER_ATTACHMENT_FILENAME = "attachment;filename=";
     private static final String CSV_FILENAME = "20220710_testApp_audit.csv";
 
     private static String CSV_CONTENT =
-        "\"APPNAME\",\"APPVERSION\",\"BUSINESSKEY\",\"ENTITY\",\"ENTITYID\",\"EVENTTYPE\",\"ID\",\"MESSAGEID\",\"PARENTPROCESSINSTANCEID\",\"PROCESSDEFINITIONID\",\"PROCESSDEFINITIONKEY\",\"PROCESSDEFINITIONVERSION\",\"PROCESSINSTANCEID\",\"SEQUENCENUMBER\",\"SERVICEFULLNAME\",\"SERVICENAME\",\"SERVICETYPE\",\"SERVICEVERSION\",\"TIME\"\n" +
-        "\"testApp\",\"\",\"\",\"{\"\"appVersion\"\":null,\"\"id\"\":\"\"10\"\",\"\"name\"\":null,\"\"processDefinitionId\"\":\"\"1\"\",\"\"processDefinitionKey\"\":null,\"\"initiator\"\":null,\"\"startDate\"\":null,\"\"completedDate\"\":null,\"\"businessKey\"\":null,\"\"status\"\":null,\"\"parentId\"\":null,\"\"processDefinitionVersion\"\":null,\"\"processDefinitionName\"\":null}\",\"\",\"PROCESS_STARTED\",\"processEventId\",\"\",\"\",\"1\",\"\",\"\",\"10\",\"0\",\"\",\"rb-my-app\",\"\",\"\",\"2022-07-07 14:59:37\"\n" +
-        "\"testApp\",\"\",\"\",\"{\"\"name\"\":\"\"var\"\",\"\"type\"\":null,\"\"processInstanceId\"\":\"\"processId\"\",\"\"value\"\":null,\"\"taskId\"\":\"\"taskId\"\",\"\"taskVariable\"\":true}\",\"var\",\"VARIABLE_CREATED\",\"variableEventId\",\"\",\"\",\"1\",\"\",\"\",\"10\",\"0\",\"\",\"rb-my-app\",\"\",\"\",\"2022-07-07 14:59:37\"\n";
+        "\"ACTOR\",\"APPNAME\",\"APPVERSION\",\"BUSINESSKEY\",\"ENTITY\",\"ENTITYID\",\"EVENTTYPE\",\"ID\",\"MESSAGEID\",\"PARENTPROCESSINSTANCEID\",\"PROCESSDEFINITIONID\",\"PROCESSDEFINITIONKEY\",\"PROCESSDEFINITIONVERSION\",\"PROCESSINSTANCEID\",\"SEQUENCENUMBER\",\"SERVICEFULLNAME\",\"SERVICENAME\",\"SERVICETYPE\",\"SERVICEVERSION\",\"TIME\"\n" +
+        "\"service_user\",\"testApp\",\"\",\"\",\"{\"\"appVersion\"\":null,\"\"id\"\":\"\"10\"\",\"\"name\"\":null,\"\"processDefinitionId\"\":\"\"1\"\",\"\"processDefinitionKey\"\":null,\"\"initiator\"\":null,\"\"startDate\"\":null,\"\"completedDate\"\":null,\"\"businessKey\"\":null,\"\"status\"\":null,\"\"parentId\"\":null,\"\"processDefinitionVersion\"\":null,\"\"processDefinitionName\"\":null}\",\"\",\"PROCESS_STARTED\",\"processEventId\",\"\",\"\",\"1\",\"\",\"\",\"10\",\"0\",\"\",\"rb-my-app\",\"\",\"\",\"2022-07-07 14:59:37\"\n" +
+        "\"service_user\",\"testApp\",\"\",\"\",\"{\"\"name\"\":\"\"var\"\",\"\"type\"\":null,\"\"processInstanceId\"\":\"\"processId\"\",\"\"value\"\":null,\"\"taskId\"\":\"\"taskId\"\",\"\"taskVariable\"\":true}\",\"var\",\"VARIABLE_CREATED\",\"variableEventId\",\"\",\"\",\"1\",\"\",\"\",\"10\",\"0\",\"\",\"rb-my-app\",\"\",\"\",\"2022-07-07 14:59:37\"\n";
```

Comment:
Let's simplify this with multiline string:
```suggestion
        """
        "ACTOR","APPNAME","APPVERSION","BUSINESSKEY","ENTITY","ENTITYID","EVENTTYPE","ID","MESSAGEID","PARENTPROCESSINSTANCEID","PROCESSDEFINITIONID","PROCESSDEFINITIONKEY","PROCESSDEFINITIONVERSION","PROCESSINSTANCEID","SEQUENCENUMBER","SERVICEFULLNAME","SERVICENAME","SERVICETYPE","SERVICEVERSION","TIME"
        "service_user","testApp","","","{""appVersion"":null,""id"":""10"",""name"":null,""processDefinitionId"":""1"",""processDefinitionKey"":null,""initiator"":null,""startDate"":null,""completedDate"":null,""businessKey"":null,""status"":null,""parentId"":null,""processDefinitionVersion"":null,""processDefinitionName"":null}","","PROCESS_STARTED","processEventId","","","1","","","10","0","","rb-my-app","","","2022-07-07 14:59:37"
        "service_user","testApp","","","{""name"":""var"",""type"":null,""processInstanceId"":""processId"",""value"":null,""taskId"":""taskId"",""taskVariable"":true}","var","VARIABLE_CREATED","variableEventId","","","1","","","10","0","","rb-my-app","","","2022-07-07 14:59:37"
        """;
```